sealed-secrets-operator/overlays/default/kustomization.yaml

```diff
@@ -1,20 +1,15 @@
 kind: Kustomization
 apiVersion: kustomize.config.k8s.io/v1beta1
-
 namespace: sealed-secrets
-
 # Include the base Sealed Secrets manifests.
```

Comment:
Remove comment

Reply:
Which comment did you have in mind? (There are two of them in this file)